easybuild/easyconfigs/s/Salmon/Salmon-0.11.2_Intel-compilers.patch

```diff
@@ -0,0 +1,31 @@
+--- CMakeLists.txt.orig	2018-08-28 10:48:05.642565702 +0200
```

Comment:
@HPC-UniOldenburg For patch files, we aim to always include a short description + either a reference to where the patch was obtained from (commit, bug issue, mailing list post, etc.) or the author of the patch.

Can you amend the patch to include this above this first line (anything above is a comment).

Keep in mind that this implies updating the SHA256 checksum for this patch file.